jkamz/ireporter
# #164327212 Update a red flag record

### What does this PR do?
Creates and endpoint for updating a redflag record

### Description of the tasks to be completed
Create an endpoint allowing a user to update a red flag record
Ensure user cannot update the status of the record

### How can this be manually tested?
Clone this repository on local machine and CD into project folder

`$ git clone https://github.com/jkamz/ireporter.git`

`$ cd ireporter`

Check out to the PR's branch
`$ git checkout ft-update-redflag-164327212`

Install app dependencies from requirements.txt

Run the following commands

`source .venv/bin/activate`
`pip install -r api/requirements.txt`
`python api/manage.py runserver`

Use postman to test the update endpoint

`localhost:8000/api/redflags/id/`

### What are the relevant Pivotal Tracker stories?
[164327212](https://www.pivotaltracker.com/story/show/164327212)

### Screenshots
#### Success
![canupdate](https://user-images.githubusercontent.com/11965186/54573962-74686e80-49ff-11e9-8d7f-f7107fbd571d.png)

#### Permissions Error
![cannotupdate](https://user-images.githubusercontent.com/11965186/54573965-792d2280-49ff-11e9-929a-60439d5d1200.png)
